imagetest/test_suites/storageperf/storage_perf_utils.go

```diff
@@ -119,6 +121,16 @@ type FIOStatistics struct {
 	X              map[string]interface{} `json:"-"`
 }
 
+// In some cases, fio might take longer to install on linux than expected.
+// To ensure tests do not fail from not finding fio, wait for the install.
+func waitFIOInstalledLinux() {
```

Comment:
It's probably a good idea to have a maximum time for this to run for before we give up and fail the test. With this, if `fio` somehow fails to get installed, we wait for the entire 45min timeout. We could definitely shorten that down to 5 minutes or less.

Reply:
It might also be more reliable (and faster, since it's not polling every 10 seconds) to switch to calling the `installFioLinux()` function so that we know exactly when it's done. That would bypass the conditions that allowed for the possibility of a race condition in the first place.

Reply:
done